Resources/views/Collector/propel.html.twig

```diff
@@ -79,12 +79,12 @@
                     <code>{{ query.sql|format_sql }}</code>
                     {% if app.request.query.has('query') and app.request.query.get('query') == i %}
                         <div class="SQLExplain">
-                        {% render controller('PropelBundle:Panel:explain', {
+                        {{ render(controller('PropelBundle:Panel:explain', {
```

Comment:
shouldn't we change the syntax here too ?

Reply:
I think no. This is because when I change the 104 line to @Propel/Panel/configuration the Symfony profiler button stops working and it throws the 500 error. In the dev.log it says: request.CRITICAL: Uncaught PHP Exception Twig\Error\RuntimeError: "An exception has been thrown during the rendering of a template ("The controller for URI "/_fragment" is not callable: Controller "@Propel/Panel/configuration" does neither exist as service nor as class.")." That is why I think that an old syntax is working. Btw, I tested everything in the context of all twig templates and config updated.